services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -71,7 +72,7 @@ tests s = testGroup "Teams LegalHold API"
     , test s "DELETE /teams/{tid}/legalhold/{uid}" testDisableLegalHoldForUser
 
       -- legal hold settings
-    , test s "POST /teams/{tid}/legalhold/settings" testCreateLegalHoldTeamSettings
+    , test s "XXX POST /teams/{tid}/legalhold/settings" testCreateLegalHoldTeamSettings
```

Comment:
is this intentional?  what does it mean?

Reply:
X-Rated 🙈